app/html/admin/editDeliverable.html

```diff
@@ -342,14 +342,6 @@
         </ons-list>
     </form>
 
-    <!-- #DURATION PICKER from https://github.com/Tartarus762/jquery-duration-picker -->
-    <script>
-        $(document).ready(function() {
```

Comment:
script tag never runs but ready isn't needed anyway since the DOM is loaded right above.